csm/conf/configure.py

```diff
@@ -50,6 +50,7 @@ async def execute(self, command):
         :param command:
         :return:
         """
+        Log.info("csm_setup: configure phase started.")
```

Comment:
csm_setup -> Setup. These are CSM setup logs. No need to mention csm_setup.

configure phase started. -> Initiating Config phase.

Reply:
Updated